ui/apps/everest/src/pages/common/empty-state/databases/index.tsx

```diff
@@ -22,7 +22,24 @@ export const EmptyStateDatabases = ({
         <EmptyStateIcon w="60px" h="60px" />
         <Box sx={centeredContainerStyle}>
           <Typography>{Messages.noDbClusters}</Typography>
-          <Typography> {Messages.createToStart}</Typography>
+          {showCreationButton ? (
+            <Typography> {Messages.createToStart} </Typography>
+          ) : (
+            <>
+              <Typography>{Messages.noPermissions}</Typography>
```

Comment:
I think we should not use `!showCreationButton` to assume there's no permissions. That prop should have only that purpose itself, which is to show/hide the button, as the name implies.
Here we are showing the "no permission" message when "showCreationButton" is false, but that's because we know the business logic from the outside. The component, however, should not be aware of it.
If we want to control the display of the "lack of permissions" message, we better have another prop for that purpose, even though both props will have the same expression to control them.